core/server/api_container/server/startosis_engine/kurtosis_types/arg_shared_helpers.go

```diff
@@ -66,3 +67,20 @@ func SafeCastToString(expectedValueString starlark.Value, argNameForLogging stri
 	}
 	return castValue.GoString(), nil
 }
+
+func SafeCastToIntegerSlice(starlarkList *starlark.List) ([]int, error) {
+	slice := []int{}
+	for i := 0; i < starlarkList.Len(); i++ {
+		value := starlarkList.Index(i)
+		starlarkCastedValue, ok := value.(starlark.Int)
+		if !ok {
+			return nil, stacktrace.NewError("An error occurred when casting element '%v' from slice '%v' to integer", value, starlarkList)
+		}
+		castedValue, ok := starlarkCastedValue.Int64()
+		if !ok {
+			return nil, stacktrace.NewError("An error occurred when casting element '%v' to Go integer", castedValue)
+		}
+		slice = append(slice, int(castedValue))
```

Comment:
is there an overflow that could happen here?

Reply:
Switched to `int64` for safety